src/server.ts

```diff
@@ -10,15 +10,27 @@ const app = new Koa();
 
 // Enable cors with default options
 app.use(cors());
-const logger = bunyan.createLogger({ name: "xdc-relayer"});
+const logger = bunyan.createLogger({ name: "xdc-relayer" });
 
 const worker = new Worker(config, logger);
 
-
 // Enable bodyParser with default options
 app.use(bodyParser());
 
 app.listen(config.port, async () => {
-  logger.info(`Server running on port ${config.port}`);
+  if (!process.env.PARENTCHAIN_WALLET_PK) {
+    logger.error("csc pk not found ,will dont running csc relayer");
+    return;
+  }
+  logger.info(`Server csc relayer running on port ${config.port}`);
   await worker.synchronization();
 });
+
+app.listen(config.port + 1, async () => {
```

Comment:
The initial idea is to run the relayer jobs using a single`worker` which spin up jobs, but looks like you decide to go with spin up multiple nodejs apps all together.
Not a big deal since we gonna rewrite this anyway

Reply:
sure , you decide this